madcars/players/pytorch_main.py

```diff
@@ -207,6 +219,8 @@ def compute_td_loss(states, actions, rewards, next_states, is_done, gamma=0.99,
             states.append(state)
             actions.append(choice)
             rewards.append(0)
+            for i in range(len(rewards)):
+                rewards[i] -= 10  # penalize for time spent not winning
```

Comment:
It looks that the longer the sequence the smaller the reward. So it requires agent to predict a game duration to predict the reward. For the reward of the first ticks it sounds weird. Does it work for you? The agent learns nothing for me.

Reply:
Oh, you're actually right. I think there's a **bug** here. What I meant is to have a way to penalize reward  in time. It would be cool to be able to accumulate the penalty over-time however, but I think it could be achieved much simpler:
~~~
rewards[-1] -= 10
~~~
This should work OK since we have a Q-value that is concerned about previous rewards (although, multiplied by `gamma`)

What's you opinon @Wovchena ?

Reply:
You can simply do `rewards.append(-10)`